angular/src/app/screens/users/components/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,26 @@
+<app-form-container>
+    <form [formGroup]="userForm" (ngSubmit)="handleSingUp()" class="form-account ">
+        <label for="firstname" class="label-account">Nombre</label>
+        <input type="text" class="input-text-account  " formControlName="first_name">
+        <span class="error-account" *ngIf="nameControl?.invalid && (nameControl?.dirty || nameControl?.touched)">Este campo es
```

Comment:
You could abstract ngIf logic to a function, and reuse that function in every span 😄